packages/backend-common/api-report.md

```diff
@@ -568,6 +568,11 @@ export type ReadUrlResponseFactoryFromStreamOptions = {
   etag?: string;
 };
 
+// @public
+export function redactLogLine(
```

Comment:
Also, one more thing that I've just thought about, maybe we should call this `redactWinstonLogLine` or something instead to namespace it a little better.